<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signup</class>
 <widget class="QWidget" name="Signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background:color #2c698d;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>330</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>User Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>390</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>490</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Phone Number</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>550</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Date of Birth</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtEmail">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>380</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:color #fff;
border-radius: 15px;
  border: 2px solid #fff;
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSignUp">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>630</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:color #1AC0C6;
border-radius: 15px;
  border: 2px solid;
  padding: 10px;
  width: 200px;
  height: 150px;</string>
      </property>
      <property name="text">
       <string>Sign Up</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtUsername">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>320</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:color #fff;
border-radius: 15px;
  border: 2px solid #fff;
</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtProfileDOB">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>560</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:color #fff;
border-radius: 15px;
  border: 2px solid #fff;
</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtProfilePhoneNo">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>500</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:color #fff;
border-radius: 15px;
  border: 2px solid #fff;
</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>191</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>440</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtProfileEmail_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>440</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:color #fff;
border-radius: 15px;
  border: 2px solid #fff;
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUploadImage">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>121</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Upload Image</string>
      </property>
     </widget>
     <widget class="QLabel" name="imageUrl">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>230</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="validationError">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>270</y>
        <width>411</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
